recipes/conan/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "conan" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "3453edc61559e4cb2d12f7d4dcfec8860ccb644dc2155b1aaccf25b6022039cc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - conan=conans.conan:run
+    - conan_server=conans.conan_server:run
+    - conan_build_info=conans.build_info.command:run
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyjwt >=1.4.0,<2.0.0
+    - requests >=2.7.0,<3.0.0
+    - colorama >=0.3.3,<0.4.0
+    - pyyaml >=3.11,<3.13.0
+    - python-patch ==1.16
+    - fasteners >=0.14.1
+    - six >=1.10.0
+    - python-node-semver ==0.2.0
+    - distro >=1.0.2,<1.2.0  # [linux]
+    - pylint >=1.8.1,<1.9.0
+    - future ==0.16.0
+    - pygments >=2.0,<3.0
+    - astroid >=1.6,<1.7
+    - bottle >=0.12.8,<0.13
+    - pluginbase >=0.5,<1.0
+  run:
+    - python
+    - pyjwt >=1.4.0,<2.0.0
+    - requests >=2.7.0,<3.0.0
+    - colorama >=0.3.3,<0.4.0
+    - pyyaml >=3.11,<3.13.0
+    - python-patch ==1.16
+    - fasteners >=0.14.1
+    - six >=1.10.0
+    - python-node-semver ==0.2.0
+    - distro >=1.0.2,<1.2.0  # [linux]
+    - pylint >=1.8.1,<1.9.0
+    - future ==0.16.0
+    - pygments >=2.0,<3.0
+    - astroid >=1.6,<1.7
+    - bottle >=0.12.8,<0.13
+    - pluginbase >=0.5,<1.0
+
+test:
+  imports:
+    - conans
+    - conans.build_info
+    - conans.client
+    - conans.client.build
+    - conans.client.cmd
+    - conans.client.conf
+    - conans.client.generators
+    - conans.client.rest
+    - conans.client.store
+    - conans.client.tools
+    - conans.model
+    - conans.search
+    - conans.server
+    - conans.server.conf
+    - conans.server.crypto
+    - conans.server.crypto.jwt
+    - conans.server.rest
+    - conans.server.rest.bottle_plugins
+    - conans.server.rest.controllers
+    - conans.server.service
+    - conans.server.store
+    - conans.test
+    - conans.test.build_helpers
+    - conans.test.command
+    - conans.test.conan_api
+    - conans.test.functional
+    - conans.test.generators
+    - conans.test.integration
+    - conans.test.model
+    - conans.test.remote
+    - conans.test.server
+    - conans.test.server.crypto
+    - conans.test.server.service
+    - conans.test.server.utils
+    - conans.test.util
+    - conans.test.utils
+    - conans.util
+  commands:
+    - conan --help
+    - conan_build_info --help
+
+about:
+  home: https://conan.io
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.md'
+  summary: Conan C/C++ package manager
+  doc_url: http://docs.conan.io/en/latest/
+  dev_url: https://github.com/conan-io/conan
+
+extra:
+  recipe-maintainers:
```

Comment:
Please get permission from the co-maintainers. If you know then personally and have had conversations offline, you can clarify that here, or you need to ping them and get their explicit consent.

Reply:
Yes, I had a conversation with these people and asked if they would be interested, and they gave their ok.